### PR TITLE
doc: Document ENI installation

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8831)

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-docs-please

**Comment by tgraf** [comment]
test-docs-please

**Comment by tgraf** [comment]
test-me-please

File: Documentation/gettingstarted/aws-eni.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+.. _k8s_aws_eni:
+
+*********************************
+Setting up Cilium in AWS ENI mode
+*********************************
+
+Create an AWS cluster
+=====================
+
+Setup a Kubernetes on AWS. You can use any method you prefer, for the simplicity
```
Comment (ianvernon) [inline Documentation/gettingstarted/aws-eni.rst]: Nit: a Kubernetes --> "Kubernetes" or "a Kubernetes cluster".


", for" --> ", but for"

File: Documentation/gettingstarted/aws-eni.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+of this tutorial, we are going to use `eksctl
+<https://github.com/weaveworks/eksctl>`_. For more details on how to set up ans
```
Comment (ianvernon) [inline Documentation/gettingstarted/aws-eni.rst]: ans --> an

File: Documentation/gettingstarted/aws-eni.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+EKS cluster using eksctl, see the section :ref:`k8s_install_eks`.
```
Comment (ianvernon) [inline Documentation/gettingstarted/aws-eni.rst]: \`\`eksctl\`\`

**Review by ianvernon — COMMENTED** [review]
LGTM apart from some grammatical / spelling / formatting nits. Feel free to merge after fixing, no need for a re-review.

**Review by joestringer — APPROVED** [review]
:+1: Looks pretty close to the instructions I came up with: https://gist.github.com/joestringer/21b351fc3253e3e95f2d28b95987d279

**Comment by tgraf** [comment]
test-docs-please

**Pull request merged**